messages/german.json

```diff
@@ -0,0 +1,1393 @@
+        "Wolfchat": ["Wolfchat Mitglied", "Wolfchat Mitglieder"],
+        "Wolfteam": ["Böser Dorfbewohner", "Böse Dorfbewohner"],
+        "Killer": ["Mörder", "Mörder"],
+        "Village": ["Dorfmitglied", "Dorfmitglieder"],
+        "Nocturnal": ["Nächtliche Rolle", "Nächtliche Rollen"],
+        "Neutral": ["Neutrale Spieler", "Neutrale Spieler"],
+        "Win Stealer": ["Siegräuber", "Siegräuber"],
+        "Hidden": ["Versteckte Rolle", "Versteckte Rollen"],
+        "Safe": ["Spezieller Dorfbewohner", "Spezieller Dorfbewohner"],
+        "Spy": ["Spion Rolle", "Spion Rollen"],
+        "Intuitive": ["intuitive Rolle", "intuitive Rollen"],
+        "Cursed": ["Verfluchter Dorfbewohner", "Verfluchte Dorfbewohner"],
+        "Innocent": ["Unschuldiger Spieler", "Unschuldiger Spieler"],
+        "Team Switcher": ["Team Wechsler", "Team Wechsler"]
+    },
+    "_phases": {
+        "*": "For assistance in configuring this metadata, please see https://werewolf.chat/Translation#Phases",
+        "join": "join",
+        "night": "Nacht",
+        "day": "Tag"
+    },
+    "no_victims": [
+        "Die Leiche eines jungen Pinguin-Haustiers wurde gefunden... Wie kann man nur so grausam sein..",
+        "Pfotenabdrücke kreisen um das Dorf.",
+        "Einige Pfotenabdrücke führen zum Dorfplatz.",
+        "Eine Pfütze von Wolfsblut und Pfotenabdrücke wurde gefunden.",
+        "Die Leiche einer getöteten Katze wurde gefunden.",
+        "Einige Haustüren wurden geöffnet.",
+        "Ein paar Häuser haben frisches Blut an den Wänden.",
+        "Ein Duft, ähnlich wie der eines Wolfes liegt in der Luft.",
+        "Halbbegrabene Wolfkots wurden gefunden.",
+        "Spuren von Wolfsfellen wurden gefunden.",
+        "Eine unheimliche Ruhe ist zu spüren.",
+        "Große Kratzspuren befinden sich an einem Baum.",
+        "Die Überreste eines der Dorfschweine wurden gefunden.",
+        "Ein paar Knochen von einem lokalen Eichhörnchen wurden gefunden.",
+        "Der Abdruck eines Wolfsgesichts dessen auf einem Pfahl geritzt wurde befindet sich in der Nähe des Dorfplatzes."
+    ],
+    "ping": [
+        "Pong!",
+        "Ping!",
+        "!gniP",
+        "!gnoP",
+        "{=pong:!}",
+        "Hallo.",
+        "Sicher.",
+        "Nein",
+        "Segmentierungsfehler",
+        "Segmentation fault (core dumped)",
+        "HTTP-Fehler 418: Ich bin eine Teekanne",
+        "🐺",
+        "Ich bin hier!",
+        "Ich weigere mich!",
+        "Was?",
+        "Meinst du nicht {=ping!command:!:bold}?",
+        "{bot_nick}.exe hat aufgehört zu arbeiten. Windows sucht nach einer Lösung für das Problem...",
+        "skynet.exe hat aufgehört zu arbeiten. Windows sucht nach einer Lösung für das Problem...",
+        "1337 Sekunden lang kein Ping empfangen.",
+        "Herzlichen glückwunsch! Sie sind die 1337. Person, die {=ping!command:!} verwendet. Sie gewinnen eine Ziege!\n{nick:@}'s Ziege geht vorbei und macht eine Kopfnuss bei {nick:@}",
```

Comment:
Glückwunsch needs to be capitalized, also i'd say "und verpasst … eine Kopfnuss"